### DISCUSSION TITLE
Get query inspector to only run desired query

### DISCUSSION THREAD

**Discussion by shuotli** [grafana/grafana#40357] [Dashboards, Panels & Visualizations]
I am making a plugin with several pickers and one query. When I refresh the query inspector, I notice all the api calls associated with the pickers and query are running. I only want to show the response for the query. Is there a way to limit which query to run?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
